data/json/monster_special_attacks/spells.json

```diff
@@ -279,5 +279,43 @@
     "field_chance": 2,
     "min_field_intensity": 2,
     "max_field_intensity": 2
+  },
+  {
+    "type": "SPELL",
+    "id": "wraith_attack",
+    "name": { "str": "Wraith Attack" },
+    "description": "Drains some of your essence.",
+    "valid_targets": [ "hostile", "ground" ],
+    "max_level": 1,
+    "flags": [ "SILENT", "NO_HANDS", "NO_LEGS" ],
+    "base_casting_time": 75,
+    "min_range": 1,
+    "max_range": 1,
+    "min_damage": -25,
+    "max_damage": -50,
+    "damage_increment": -10.0,
+    "shape": "blast",
+    "effect": "recover_energy",
+    "effect_str": "FATIGUE"
+  },
+  {
+    "type": "SPELL",
+    "id": "impossible_shape_attack",
+    "name": { "str": "impossible Shape Attack" },
+    "description": "Scares you.",
+    "valid_targets": [ "hostile", "ground" ],
+    "max_level": 1,
+    "flags": [ "SILENT", "NO_HANDS", "NO_LEGS" ],
+    "base_casting_time": 75,
+    "min_range": 1,
+    "max_range": 5,
+    "shape": "blast",
+    "effect": "effect_on_condition",
+    "effect_str": "EOC_impossible_shape_attack"    
+  },
+  {
+    "type":"effect_on_condition",
+    "id":"EOC_impossible_shape_attack",
+    "effect": { "u_add_effect": "nightmares", "duration": "12 hours" }
```

Comment:
Does this have any effect if the person doesn't sleep for the next 12 hours?

Reply:
No, but it can stack up till 3 days, and the wraiths will potentially be making you more tired.